pkg/cmd/marshal.go

```diff
@@ -41,6 +42,11 @@ func (c *Marshaler) AddFlags(cmd *cobra.Command) {
 		"as-json",
 		false,
 		"Prints value as json")
+	cmd.Flags().BoolVar(
```

Comment:
It's probably time to switch to `--output` instead of the various ordered output flags. We can leave the old ones in as deprecated/hidden.

Reply:
Done.